RxSwift/ObservableType+Extensions.swift

```diff
@@ -130,8 +130,8 @@ extension ObservableType {
 import Foundation
 
 extension Hooks {
-    public typealias DefaultErrorHandler = (_ subscriptionCallStack: [String], _ error: Error) -> Void
-    public typealias CustomCaptureSubscriptionCallstack = () -> [String]
+    public typealias DefaultErrorHandler = @Sendable (_ subscriptionCallStack: [String], _ error: Error) -> Void
```

Comment:
defining
```swift
public static var defaultErrorHandler: sending DefaultErrorHandler {
```
gives the error `'sending' may only be used on parameters and results` so it can't be added here.

assigning `defaultErrorHandler` from a main actor isolated context, without marking the closure as Sendable, and calling it from a non-main thread, would cause a runtime crash in Swift 6.

```swift
@MainActor
func test() {
    Hooks.defaultErrorHandler = { _, _ in }
}
test()
DispatchQueue.global().async(execute: { () in Hooks.defaultErrorHandler([], RxError.unknown) })
```

So it is safer to have the DefaultErrorHandler closure as sendable